earth_enterprise/integration_test/step_impl/assets.py

```diff
@@ -0,0 +1,420 @@
+from getgauge.python import step
+import os
+import subprocess
+import time
+
+BASE_ASSET_PATH = "gauge_tests"
+DATABASE_PATH = os.path.join(BASE_ASSET_PATH, "Databases")
+MAP_LAYER_PATH = os.path.join(BASE_ASSET_PATH, "MapLayers")
+IMAGERY_PROJECT_PATH = os.path.join(BASE_ASSET_PATH, "Projects", "Imagery")
+TERRAIN_PROJECT_PATH = os.path.join(BASE_ASSET_PATH, "Projects", "Terrain")
+VECTOR_PROJECT_PATH = os.path.join(BASE_ASSET_PATH, "Projects", "Vector")
+MAP_PROJECT_PATH = os.path.join(BASE_ASSET_PATH, "Projects", "Map")
+IMAGERY_RESOURCE_PATH = os.path.join(BASE_ASSET_PATH, "Resources", "Imagery")
+TERRAIN_RESOURCE_PATH = os.path.join(BASE_ASSET_PATH, "Resources", "Terrain")
+VECTOR_RESOURCE_PATH = os.path.join(BASE_ASSET_PATH, "Resources", "Vector")
+
+def get_asset_root():
+  return os.getenv('ge_asset_root')
+
+def get_src_data_path():
+  return os.getenv('ge_src_data_path')
+
+def make_dirs(dirs):
+  try:
+    os.makedirs(dirs)
+  except OSError:
```

Comment:
I don't see how this code makes sure the exception was raised because the directory exists.  To do that, you'd use code like the following: 

```suggestion
  except OSError as exc:
      if exc.errno != errno.EEXIST:
          raise
```